multisig/src/main.rs

```diff
@@ -40,17 +42,18 @@ struct Multisig {
 }
 
 impl Multisig {
-    pub fn new(_ctx: &Context, owners: Set<Address>, required: u32) -> Self {
+    pub fn new(_ctx: &Context, addresses: Vec<String>, required: u32) -> Self {
```

Comment:
```suggestion
    pub fn new(_ctx: &Context, addresses: &[Address], required: u32) -> Self {
```
not sure if serde'll let you deserialize a slice of addresses. Use a `Vec`, if not.

also, is there a better name for `required`? maybe `required_signature`?